### PR TITLE
Noto Sans SignWriting: Version 2.004; ttfautohint (v1.8.4.7-5d5b) added

### LINKED ISSUE #3628: Add Noto Sans SignWriting
https://github.com/googlefonts/noto-fonts/tree/main/unhinted/ttf/NotoSansSignWriting

### PR DESCRIPTION
**Pull request by simoncozens**
5de8779: [gftools-packager] Noto Sans SignWriting: Version 2.004; ttfautohint (v1.8.4.7-5d5b) added

* Noto Sans SignWriting Version 2.004; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/sign-writing.git at commit https://github.com/notofonts/sign-writing/commit/.

Fixes #3628

### CODE REVIEW

**Comment by simoncozens** [comment]
New font, needs article and description @twardoch.

File: ofl/notosanssignwriting/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name: "Noto Sans SignWriting"
```
Comment (RosaWagner) [inline ofl/notosanssignwriting/METADATA.pb:1]: Are we sure about SignWriting? not Sign Writing, and not Signwriting. If yes, then it should be added to the camel case exception list: https://github.com/googlefonts/fontbakery/blob/main/Lib/fontbakery/data/googlefonts/camelcased_familyname_exceptions.txt
cc @davelab6 @chrissimpkins

Comment (simoncozens) [reply ofl/notosanssignwriting/METADATA.pb:1]: The official name of the script is, indeed, SignWriting.

Comment (RosaWagner) [reply ofl/notosanssignwriting/METADATA.pb:1]: CamelCase accepted, the name should be added to the exception list in fontbakery

File: ofl/notosanssignwriting/METADATA.pb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  copyright: "Copyright 2022 The Noto Project Authors (https://github.com/notofonts/sign-writing)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/sign-writing.git"
+  archive_url: "https://github.com/notofonts/sign-writing/releases/download/NotoSansSignWriting-v2.004/NotoSansSignWriting-v2.004.zip"
+}
+is_noto: true
```
Comment (RosaWagner) [inline ofl/notosanssignwriting/METADATA.pb:22]: Should there be subset, language, sample text defined for this?

Comment (simoncozens) [reply ofl/notosanssignwriting/METADATA.pb:22]: I'll add some sample text.

Comment (simoncozens) [reply ofl/notosanssignwriting/METADATA.pb:22]: Actually it looks like the font is not working properly anyway.

**Comment by simoncozens** [comment]
Updated

## Noto Sans SignWriting: Version 2.004; ttfautohint (v1.8.4.7-5d5b) added

---
 4da2951: [gftools-packager] Noto Sans SignWriting: Version 2.004; ttfautohint (v1.8.4.7-5d5b) added

* Noto Sans SignWriting Version 2.004; ttfautohint (v1.8.4.7-5d5b) taken from the upstream repo https://github.com/notofonts/sign-writing.git at commit https://github.com/notofonts/sign-writing/commit/.

**Comment by RosaWagner** [comment]
@simoncozens couldn't find a subset definition in glyphset repo for SignWriting, I think we would need to add one before merging otherwise all the signs will be subsetted from the font

**Pull request merged**